content-scripts/components/HeaderAuthentication.tsx

```diff
@@ -0,0 +1,138 @@
+import jsx from "texsaur";
+import { AuthSession } from "../types";
+import { togglePopover } from "../modules/utilities/popover";
+
+interface Props {
+  auth: AuthSession | null;
+}
+
+const Authentication = ({ auth }: Props) => {
+  if (auth)
+    return (
+      <div id="se-auth">
+        <button
+          id="se-auth-notifications-button"
+          className={`se-button se-icon-button ${auth.hasNotifications ? "se-badge" : ""
+            }`}
+          onclick={() => togglePopover("se-auth-notifications-menu")}
+        >
+          <span className="se-icon ri-notification-line"></span>
+        </button>
+        <div id="se-auth-notifications-menu">
+          <input
+            type="radio"
+            name="se-auth-notifications"
+            id="se-auth-new-notifications-radio"
+            checked
+          />
+          <input
+            type="radio"
+            name="se-auth-notifications"
+            id="se-auth-read-notifications-radio"
+          />
+          <div id="se-auth-notifications-header">
+            <label htmlFor="se-auth-new-notifications-radio">Novas</label>
+            <label htmlFor="se-auth-read-notifications-radio">Lidas</label>
+          </div>
+          <div id="se-auth-notifications-list">
+            <ol id="se-auth-new-notifications"></ol>
+            <ol id="se-auth-read-notifications"></ol>
+            <div id="se-auth-empty-notifications">
+              <span className="se-icon ri-notification-off-line"></span>
+              <span>Sem notificações</span>
+            </div>
+            <div className="se-loading-indicator">
+              <span className="se-icon ri-refresh-line"></span>
+            </div>
+          </div>
+        </div>
+
+        <button id="se-auth-profile-button" onclick={() => togglePopover("se-auth-profile-menu")}>
+          <img
+            src={`fotografias_service.foto?pct_cod=${auth.number}`}
+            alt="Foto de perfil"
+          />
+        </button>
+        <div id="se-auth-profile-menu">
+          <div id="se-auth-header">
+            <span>{auth.name}</span>
+            <span>{auth.number}</span>
+          </div>
+          <nav id="se-auth-profile-links">
+            <a
+              className="se-auth-profile"
+              href={`fest_geral.cursos_list?pv_num_unico=${auth.number}`}
+            >
+              <span className="se-icon ri-user-line"></span> Perfil
+            </a>
+            <a
+              href={`gpag_ccorrente_geral.conta_corrente_view?pct_cod=${auth.number}`}
+            >
+              <span className="se-icon ri-money-euro-circle-line"></span> Conta
+              corrente
+            </a>
+            <a
+              id="se-logout-button"
+              href="vld_validacao.sair?p_address=WEB_PAGE.INICIAL"
+            >
+              <span className="se-icon ri-logout-box-line"></span> Terminar
```

Comment:
<Icon /> component?

Reply:
That would be good, but we should check out how we are currently doing the icons, there may be an easy way of having an `<Icon name=""/>` component.

Reply:
Every custom icon is a span with `se-icon` class

Reply:
I have changed all the `span` to` <Icon />`, check it out. Now is atom can improve in the future and be ease to change, in my opinion